src/LuaScriptBuilder.php

```diff
@@ -237,7 +237,7 @@ private function quoteValue($str)
             return $str;
         }
 
-        return "'" . addcslashes($str, "\000\n\r\\\032\047") . "'";
+        return "'" . addcslashes($str, "\000\n\r\\\047") . "'";
```

Comment:
`\032` octal is a [substitute character](https://en.wikipedia.org/wiki/Substitute_character). It's not a simple one so this needs to be checked if there's any threat in not escaping it.